src/Pos/WorkMode.hs

```diff
@@ -72,6 +73,7 @@ modeContext [d|
         !(PeerStateTag  ::: PeerStateCtx Production)
         !(JsonLogConfig ::: JsonLogConfig)
         !(LoggerName    ::: LoggerName)
+        !(RelayContext  ::: RelayContext Production)
```

Comment:
Oh, btw, I see that `PeerStateCtx` has `Production` too, but I guess `PeerStateCtx` is not used currently, so it's just a leftover.